media/css/firefox/whatsnew/whatsnew-lite.scss

```diff
@@ -64,41 +64,77 @@
 
 .lite-container {
     display: flex;
-    align-items: center;
+    align-items: flex-end;
 
     @media #{$mq-lg} {
-        margin-left: 80px;
+        min-height: 200px;
+        max-width: 500px;
+        h1 {
+            margin-bottom: 20px;
```

Comment:
We try and use standardized variables for things like spacing. Something like `$spacing-lg` is probably a good match here. You can see a complete list of tokens available in Protocol here: https://protocol.mozilla.org/fundamentals/tokens.html